### PR TITLE
Add explicit PlainPubKey algorithm

### LINKED ISSUE #78: Add "PlainPubKey"
see https://github.com/jonasnick/bips/pull/74#discussion_r1011834324

### PR DESCRIPTION
**Pull request by jonasnick**
Fixes #78. I didn't change `Sign` to use `PlainPubKey` because it looks a bit ugly (it's extremely redundant).

### CODE REVIEW

File: bip-musig2.mediawiki [hunk 1]
```diff
@@ -249,6 +249,17 @@ The following conventions are used, with constants as defined for [https://www.s
 
 === Key Generation ===
 
+==== Key Generation of participating Signers ====
```
Comment (real-or-random) [inline bip-musig2.mediawiki]:
```suggestion
==== Key Generation of an Individual Signer ====
```

File: bip-musig2.mediawiki [hunk 2]
```diff
@@ -249,6 +249,17 @@ The following conventions are used, with constants as defined for [https://www.s
...
+
+<div>
+Algorithm ''PlainPubKey(sk)'':
```
Comment (real-or-random) [inline bip-musig2.mediawiki]:
```suggestion
Algorithm ''IndividualPubKey(sk)'':
```

Can we call it like this? I think "plain" is somewhat confusing, we use it to distinguish from "x-only", but that distinction is orthogonal to what we add here. For example,  we also have "GetPlainPubKey()", which refers to the *aggregate* pubkey in plain format.

---

We could add a footnote that this matches PubKey from BIP340.

**Review by real-or-random — COMMENTED** [review]
ACK mod nits

I agree that it's better not to use it in Sign.

File: bip-musig2.mediawiki (hunk 2 above)
Comment (jonasnick) [reply bip-musig2.mediawiki]: Do you mean add a footnote that this does _not_ match BIP340's PubKey?

Comment (real-or-random) [reply bip-musig2.mediawiki]: Oh, I actually meant that it matches. But yeah, sorry, brain fart. :)

Maybe we could footnote instead that it matches the plain key generation traditionally used for ECDSA in Bitcoin (if you agree with removing the "PlainPubKey" name.)

Comment (jonasnick) [reply bip-musig2.mediawiki]: Ah that's a good idea, done. Also replaced  `plain_pk_gen` with `individual_pk` in the python code.

File: bip-musig2.mediawiki (hunk 1 above)
Comment (jonasnick) [reply bip-musig2.mediawiki]: done

**Review by real-or-random — APPROVED** [review]
ACK
